legal-api/tests/unit/resources/v2/test_business_parties.py

```diff
@@ -232,16 +239,18 @@ def test_get_business_ceased_parties(session, client, jwt):
             datetime.datetime(2013, 5, 17),
             EntityRole.RoleTypes.director,
         )
-        legal_entity.entity_roles.append(party_role)
+        legal_entity.entityty_roles.append(party_role)
```

Comment:
Is it a typo?

Reply:
`entityty_roles` is a typo isn't it?

Reply:
guys, i think entityty_roles is a typo, i might be wrong though idk, hahahha

Reply:
haha